packages/react/src/runtimes/local/LocalThreadListRuntimeCore.tsx

```
@@ -3,206 +3,211 @@ import { ThreadListRuntimeCore } from "../core/ThreadListRuntimeCore";
-      }
-      if (state === "deleted") {
-        data.dispose();
+      case "deleted":
         this._threadData.delete(threadId);
-      }
+        data.dispose();
+        break;
 
-      data.lastState = state;
+      default:
+        const _exhaustiveCheck: never = newState;
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)